sources/functions/utils

```diff
@@ -6,10 +6,14 @@ function install_rar() {
     if ! which unrar > /dev/null; then
         case "$(_os_arch)" in
             "amd64")
-                apt_install rar unrar
+                apt_install rar unrar || {
+                    exit 1
```

Comment:
This is superfluous, no? Doesn't apt already exist the current installer on error?

Reply:
Oh yeah, should be